src/BatchReader/BatchReader.ts

```diff
@@ -73,6 +76,9 @@ export class BatchReader {
     this.container.bind<Service>('Service').to(Service)
     this.container.bind<ServiceConfiguration>('ServiceConfiguration').toConstantValue({
       readNextDirectoryIntervalInSeconds: this.configuration.readNextDirectoryIntervalInSeconds,
+      batchReaderReadNextDirectoryRequest: this.configuration.batchReaderReadNextDirectoryRequest,
```

Comment:
Shouldn't we be using the `ExchangeConfiguration` for this instead?

Reply:
I'm using from ExchangeConfiguration in the next line, it is not being used. I will remove